libraries/redcore/api/base.php

```diff
@@ -124,6 +124,21 @@ public function setOptions($options = null)
 	}
 
 	/**
+	 * Transform a source field data value.
+	 *
+	 * Creates a custom response text error the given errors
+	 *
+	 * @param   string  $errorCode  HTTP error code
+	 * @param   array   $errors     Array with errors to be set in the message
+	 *
+	 * @return string
+	 */
+	public function createCustomHttpError($errorCode, $errors)
+	{
+		return self::$statusTexts[$errorCode] . ': ' . implode(chr('. '), $errors);
```

Comment:
check here if $errors is empty then return just self::$statusTexts[$errorCode] (so it gets returned without : ) otherwise you use this line...